### DISCUSSION TITLE
Desktop Wallet List (is it Limited to 10 )

### DISCUSSION THREAD

**Discussion by Hyderali1723** [WalletConnect/walletconnect-monorepo#1157] [Developer Support]
After integrating a Standalone Client process in react application,  i 'm  getting only  10 wallets in desktop tab , as opposed to 170 wallets as displayed in documentation , any suggestion how to fetch all wallet list ?

**Comment by chadyj** [comment]
The modal shows different wallets depending on the context.
- On mobile, it will show many
- On desktop, only a smaller amount are shown.

Try opening the mobile on mobile to see if results differ.
